src/HtmlExtension.php

```diff
@@ -35,6 +35,7 @@ public function getFunctions()
     {
         return [
             new TwigFunction('html_classes', 'twig_html_classes'),
+            new TwigFunction('html_attr', [$this, 'twig_html_attributes'], ['is_safe' => ['html']]),
```

Comment:
this is broken. The callable is not a method on `$this` but a function in your implementation.

Reply:
fixed